src/sbm.jl

```diff
@@ -78,6 +78,12 @@
     temperature::Vector{T} | "°C"
     # Potential evapotranspiration [mm Δt⁻¹]
     potential_evaporation::Vector{T}
+    # Multiplication factor [-] for epot
+    epot_factor::Vector{T} | "-"
+    # Multiplication factor [-] for precip
+    precip_factor::Vector{T} | "-"
+    # Additive factor [-] for temp
```

Comment:
Additive factor -> offset. And units are °C.

Reply:
good point, changed in 1a9c2c43cfc531832f5c56f4be854853992b03f4